### ISSUE TITLE
_this._aniStartHighlight is not a function

### ISSUE THREAD

**Issue by Secretmapper** [xinthink/react-native-material-kit#188]
For some weird reason I'm getting a `_this._aniStartHighlight is not a function` error. I'm on RN 0.26.3

<img width="459" alt="screen shot 2016-06-29 at 2 05 58 pm" src="https://cloud.githubusercontent.com/assets/4077143/16442147/a61a6c78-3e02-11e6-9272-d7230398a15d.png">

Has anyone encountered this before? Is this an RN transpilation error?

**Comment by Thomsos** [comment]
I'm getting this error as well on RN 0.28

**Comment by Crash--** [comment]
Can you provide a reproductible exemple? I've tried with RN 0.26, 0.28 and 0.29 and I didn't manage to reproduce this bug :/

**Comment by Crash--** [comment]
@Secretmapper @Thomsos Do you still have this issue? If yes, can you provide an exemple to reproduce this issue?

**Comment by Thomsos** [comment]
Yes i still have this issue. But I believe it has something to do with how the function is defined using class properties and my environment not supporting it.

` _onFocus = (e) => {
    this._aniStartHighlight(true);
    this._callback('onFocus', e);
  };`

**Comment by Crash--** [comment]
I don't think that it's the only place when we use ES2015 feature. Can you provide a code snippet you use?
